### PR TITLE
adds option to not send email notifications on course transitions

### PR DESCRIPTION
**Pull request by pitwegner**

### CODE REVIEW

File: evap/staff/templates/staff_course_operation.html [hunk 1]
```diff
@@ -22,7 +22,13 @@
             {% endfor %}
             </tbody>
         </table>
-
+        {% if no_email_btn %}
+        <div class="checkbox">
```
Comment (janno42) [inline evap/staff/templates/staff_course_operation.html]: please indent all lines inside the if

File: evap/staff/templates/staff_course_operation.html [hunk 2]
```diff
@@ -22,7 +22,13 @@
...
+            <label>
+                <input type="checkbox" name="send_email" checked> {% trans "Send Email notification" %}
```
Comment (janno42) [inline evap/staff/templates/staff_course_operation.html]: `Send email notifications`

File: evap/staff/views.py [hunk 3]
```diff
@@ -132,11 +132,11 @@ def semester_course_operation(request, semester_id):
         if operation == 'revertToNew':
             helper_semester_course_operation_revert(request, courses)
         elif operation == 'prepare' or operation == 'reenableEditorReview':
-            helper_semester_course_operation_prepare(request, courses)
+            helper_semester_course_operation_prepare(request, courses, request.POST.get('send_email'))
         elif operation == 'approve':
             helper_semester_course_operation_approve(request, courses)
         elif operation == 'publish':
-            helper_semester_course_operation_publish(request, courses)
+            helper_semester_course_operation_publish(request, courses, request.POST.get('send_email'))
```
Comment (karyon) [inline evap/staff/views.py]: please put `request.POST.get('send_email') == 'on'` into a variable and then hand over only a boolean

File: evap/staff/views.py [hunk 4]
```diff
@@ -176,6 +176,7 @@ def semester_course_operation(request, semester_id):
         operation=operation,
         current_state_name=current_state_name,
         new_state_name=new_state_name,
+        no_email_btn=operation in ['prepare', 'reenableEditorReview', 'publish']
```
Comment (karyon) [inline evap/staff/views.py]: the button is a checkbox now, could you call that `show_email_checkbox` or something?

**Pull request merged**